input/fsh/Profiles clinical datapoints/TumorStaging.fsh

```diff
@@ -0,0 +1,184 @@
+* value[x] from TNMRegionalNodesVS (preferred)
+* value[x] MS
+* effectiveDateTime MS
+
+Instance: PathologicalTNMNodalStagePatient147
+InstanceOf: PathologicalTNMNodalStage
+Description: "Example showing TNM staging for nodal stage"
+* status = #final "final"
+* subject = Reference(BreastCancerPatient147)
+* valueCodeableConcept = AJCC#pN0
+* method = SCT#897275008 "American Joint Commission on Cancer, Cancer Staging Manual, 8th edition neoplasm staging system (tumor staging)"
+
+Mapping: PathologicalTNMNodalStageToICHOM
+Source:	PathologicalTNMNodalStage
+Target: "https://connect.ichom.org/patient-centered-outcome-measures/breast-cancer"
+Id: PathologicalTNMNodalStageMapping
+Title: "Pathological TNM nodal stage to ICHOM set"
+Description: "Mapping of the pathological TNM nodal stage to the ICHOM breast cancer PCOM set." 	
+* valueCodeableConcept -> "Pathological nodal stage"
+
+
+
+// CLINICAL DISTANT METASTASES STAGE
+Profile:  ClinicalTNMDistantMetastases
+Id: clinical-tnm-distant-metastases
+Parent: Observation
+Title: "Clinical TNM distant metastases"
+Description: "Represents the clinical stage of the extent of a tumor metastasis in remote anatomical locations (per AJCC 8th Ed.)"
+* insert PublicationProfileRuleset
+* code = SCT#385380006 "Metastasis category finding"
+* subject only Reference(BreastCancerPatient)
+* value[x] only CodeableConcept 
+* value[x] from TNMDistantMetastasesVS (preferred)
+* value[x] MS
+* effectiveDateTime MS
+
+Instance: ClinicalTNMDistantMetastasesPatient147
+InstanceOf: ClinicalTNMDistantMetastases
+Description: "Example showing TNM staging for distant metastases"
+* status = #final "final"
+* subject = Reference(BreastCancerPatient147)
+* valueCodeableConcept = AJCC#cMX
+* method = SCT#897275008 "American Joint Commission on Cancer, Cancer Staging Manual, 8th edition neoplasm staging system (tumor staging)"
+
+Mapping: ClinicalTNMDistantMetastasesToICHOM
+Source:	ClinicalTNMDistantMetastases
+Target: "https://connect.ichom.org/patient-centered-outcome-measures/breast-cancer"
+Id: ClinicalTNMDistantMetastasesMapping
+Title: "TNM score of clinical distant metastases to ICHOM set"
+Description: "Mapping of the TNM score of the clinical distant metastases to the ICHOM breast cancer PCOM set." 	
+* valueCodeableConcept -> "Clinical distant metastasis"
+
+// PATHOLOGICAL DISTANT METASTASES STAGE
+Profile:  PathologicalDistantMetastasesTNM
+Id: pathological-tnm-distant-metastases
+Parent: Observation
+Title: "Pathological TNM distant metastases"
+Description: "Represents the pathological stage of the extent of a tumor metastasis in remote anatomical locations (per AJCC 8th Ed.)"
+* insert PublicationProfileRuleset
+* code = SCT#385380006 "Metastasis category finding"
+* subject only Reference(BreastCancerPatient)
+* value[x] only CodeableConcept 
+* value[x] from TNMDistantMetastasesVS (preferred)
+* value[x] MS
+* effectiveDateTime MS
+
+Instance: PathologicalDistantMetastasesTNMPatient147
```

Comment:
Shouldn't it also include the code and the effectiveDateTime? In my opinion it is important to provide complete examples.

* code = SCT#385380006 "Metastasis category finding"